client/verta/verta/_deployment/endpoint.py

```diff
@@ -227,6 +237,10 @@ def update_from_config(self, filepath):
         return self._update_from_dict(update_dict)
 
     def _update_from_dict(self, update_dict):
+        if (not "model_version_id" in update_dict and not "run_id" in update_dict) or \
+                ("model_version_id" in update_dict and "run_id" in update_dict):
```

Comment:
Actually, here's an alternative that I personally find easier to read
```suggestion
        id_keys = ('model_version_id', 'run_id')
        if (all(key in update_dict for key in id_keys)
                or not any(key in update_dict for key in id_keys)):
```

Reply:
Resolved below.